src/test/java/seedu/address/logic/commands/RsvpCommandTest.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalEvents.ACADEMIC;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.event.Description;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.FromDate;
+import seedu.address.model.event.ToDate;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.rsvp.Rsvp;
+import seedu.address.model.rsvp.RsvpStatus;
+
+public class RsvpCommandTest {
+
+    private Model model;
+
+    @BeforeEach
+    public void initialize() {
+        model = new ModelManager();
+    }
+
+    @Test
+    public void constructor_nullRsvp_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new RsvpCommand(null, null, null));
+    }
+
+    @Test
+    public void execute_nullRsvp_throwsCommandException() {
+        Index invalidIndex = Index.fromZeroBased(0);
+        RsvpCommand rsvpCommand = new RsvpCommand(invalidIndex, invalidIndex, RsvpStatus.CC);
+        assertThrows(CommandException.class, () -> rsvpCommand.execute(model));
+    }
+
+    @Test
+    public void execute_invalidRsvp_throwsCommandException() {
```

Comment:
Perhaps you could rename this test case to execute_personNotInEvent_throwsCommandException to reflect that we are testing that we cannot create an Rsvp for a person who is not in the event